wwrmi/api.go

```diff
@@ -838,10 +836,11 @@ func (client *Client) postLogs() (err error) {
 
 	if err == nil {
 		resp, err = client.client.Do(req)
-		// close response body in case of success and errors, not required if no response
-		if resp != nil {
-                        defer resp.Body.Close()
-                }
+		if err != nil {
```

Comment:
Should we collapse these nested double checks? Are we not supposed to Close if err != Nil.
Also if we have an error can we add an error msg/print.

Reply:
No err and resp are separate. we can have non nil response even in case of error